Source/Core/Core/HW/WiimoteEmu/WiimoteEmu.cpp

```diff
@@ -429,18 +443,31 @@ void Wiimote::SendDataReport()
     // Acceleration:
     if (rpt_builder.HasAccel())
     {
+      Common::Vec3 acceleration;
+      if (m_motionsource == MotionSource::IMU)
+        acceleration = m_imu_accelerometer->GetState();
```

Comment:
Applying the configured sideways/upright orientation allows games like Mario Kart Wii to be played without having to remap everything for a sideways remote. `GetOrientation() * m_imu_accelerometer->GetState()`.

I'd do the same for gyro below: `GetOrientation() * m_imu_gyro->GetState()`.

Reply:
Doing like you suggest. I don't have Mario Kart available but I think I can use New Super Mario's tilt lifts to test.

Reply:
Testet sideways setting with New Super Mario Bros Wii for accelerometer and Wii Sports Resort Skydiving for gyro. Haven't tested upright setting.